StaticCache.php

```diff
@@ -0,0 +1,38 @@
+<?
+require_once '../Matryoshka/library/iFixit/Matryoshka/Backend.php';
```

Comment:
Since we *only* use `$backend->get()` and `$backend->set()` in our implementation, I'd vote for leaving the specifics of the backend to the library user and not force them use Matryoshka (even though it's great).